### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by nicholascok**
just a few bug fixes 😎, nothing to concern yourself with. you probably don't even need to look at the hand- i mean han- i mean ha- i mean ran- i mean nutel- i mean code, code - i mean han- code- i mean- code. code.

### CODE REVIEW

**Comment by SmokeyTheKat** [comment]
Good commit 👍, waiting for one more reviewer to review before merge.

**Comment by nicholascok** [comment]
no, you probably don't need anyone else, you're your own guy right? you don't need the approval of others. take a stance against the system. break the simulation. let this one slide 😎, if you know what I mean.

**Comment by SmokeyTheKat** [comment]
Ok. Done.

File: fish/fish18.fish [hunk 1]
```diff
@@ -0,0 +1,19 @@
+    ________
```
Comment (SmokeyTheKat) [inline fish/fish18.fish:1]: nutella is bad for you

**Comment by SmokeyTheKat** [comment]
after review, we have concluded that this pr is not suited for this repo.

**Comment by nicholascok** [comment]
this is outrageous! unfair! unjust! you can't do that!

**Comment by nicholascok** [comment]
I wish to speak to the maître du code, someone who can compete with my sheer puissance.

**Comment by nicholascok** [comment]
for the record.

**Pull request closed without merging**